modules/flowable-ui/flowable-ui-app/src/main/java/org/flowable/ui/application/CDTaskListener.java

```diff
@@ -0,0 +1,94 @@
+package org.flowable.ui.application;
+
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.flowable.engine.delegate.TaskListener;
+import org.flowable.task.service.delegate.DelegateTask;
+import org.flowable.variable.api.persistence.entity.VariableInstance;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CDTaskListener
+  implements TaskListener
+{
+  public ProducerFactory<String, String> producerFactory()
+  {
```

Comment:
The format of this file is very different then the previous one which better matches the project's style.